toolbox

```diff
@@ -1040,9 +1041,11 @@ list_images()
         return 1
     fi
 
-    if ! output=$(echo "$details" \
-                  | sed "s/ \{2,\}/\t/g" 2>&3 \
-                  | column --separator "$tab" --table --table-columns "IMAGE ID,IMAGE NAME,CREATED" 2>&3); then
+    table_data=$(printf "%s\t%s\t%s\n" "IMAGE ID" "IMAGE NAME" "CREATED"; \
```

Comment:
We should skip all this when `details` is an empty string. This was currently being taken care of by `column` but not anymore.